src/components/subscriptions/addons/edit/AddonRatesEditor.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Form fields for adding, editing, and deleting app addon rates.
+ *
+ * @author sarahr
+ */
+
+import React from "react";
+import { useTranslation } from "i18n";
+import {
+    Button,
+    TableContainer,
+    Table,
+    TableHead,
+    TableRow,
+    TableCell,
+    TableBody,
+    Typography,
+} from "@mui/material";
+import {
+    FormNumberField,
+    FormTimestampField,
+} from "components/forms/FormField";
+import buildID from "components/utils/DebugIDUtil";
+import { minValue, nonEmptyField } from "components/utils/validations";
+import { Field } from "formik";
+import { Add, Delete } from "@mui/icons-material";
+
+import ids from "../../ids";
+
+function AddonRateEditorRow(props) {
+    const { baseId, fieldName, key, onDelete } = props;
```

Comment:
Note that the GitHub Action is now reporting a warning here (which should also be reported if you run `npm run check` locally):

> 'key' is assigned a value but never used

Reply:
Sounds good, thanks. I think I have that fixed locally, but I haven't pushed that change yet because I ran into some errors during testing after making some recent changes.